src/components/Contact/Contact.tsx

```diff
@@ -67,13 +70,12 @@ export const Contact = ({
           {contactRole}
         </RichText>
       )}
-      {institution && (
-        <p className="cc-contact__institution" itemProp="worksFor">
-          {institutionCountry
-            ? `${institution}, ${institutionCountry}`
-            : institution}
-        </p>
-      )}
+      {institutions &&
+        institutions.map(e => (
```

Comment:
Minor: you could use object destructuring here to access the keys directly i.e.

```js
institutions.map(({ country, name }) => (
  <p className="cc-contact__institution" itemProp="worksFor">
    {country ? `${name}, ${country}` : name}
  </p>
))}
```

Reply:
Done ! Note I had to alias `name` to `institutionName` as otherwise it shadowed an outer variable.